powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/CreateApplicationTest.java

```diff
@@ -53,7 +53,7 @@ public void testCreateApplicationWithDuplicateName() {
         CreateApplicationRequest request = new CreateApplicationRequest();
         request.setApplicationId(testId);
         assertDoesNotThrow(()-> powerAuthService.createApplication(request));
-        assertThrows(DataIntegrityViolationException.class, ()-> powerAuthService.createApplication(request));
+        assertThrows(GenericServiceException.class, ()-> powerAuthService.createApplication(request));
```

Comment:
Are we really using default formatter? Because in that case it should be `() ->`

Reply:
Great! If we are already touching the line, why not to make it perfect? ;)